mentors/c/e.json

```
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "enamcse",
+    "name": "Enamul Hassan",
+    "link_text": null,
+    "link_url": null,
+    "avatar_url": null,
+    "bio": "I started Programming with C and loved it. Later, I coded in C++ and learned Python. But the affection to C still in my heart."
```

```suggestion
    "bio": "I started Programming with C and loved it. Later, I coded in C++ and learned Python. But the affection to C is still in my heart."
```

Feels like a word is forgotten there.  Otherwise looks good.